### PR TITLE
 [MIG][11.0] account_payment_batch_process

### PR DESCRIPTION
**Pull request by murtuzasaleh**
Batch Payments Processing
=====================
This module allows to process payments in batch for customers invoices and vendor bills.

### CODE REVIEW

**Comment by murtuzasaleh** [comment]
@max3903 Can you please review this PR.

File: account_payment_batch_process/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2017 Ursa Information Systems <http://www.ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Batch Payments Processing",
+    "summary": "Process Payments in Batch",
+    "version": "11.0.1.0.1",
```
Comment (max3903) [inline account_payment_batch_process/__manifest__.py]:
```suggestion
    "version": "12.0.1.0.0",
```

File: account_payment_batch_process/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "category": "Generic Modules/Payment",
+    "website": "https://github.com/OCA/account-payment",
+    "depends": [
+        "account_check_printing",
+    ],
+    "data": [
+        "wizard/invoice_batch_process_view.xml",
+        "views/invoice_view.xml"
+    ],
+    "installable": True,
```
Comment (max3903) [inline account_payment_batch_process/__manifest__.py]:
```suggestion
    "application": False,
    "development_status": "Beta",
    "maintainers": ["max3903"],
```

File: account_payment_batch_process/models/account_payment.py [hunk 3]
```diff
@@ -0,0 +1,109 @@
+# Copyright 2017 Ursa Information Systems <http://www.ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
```
Comment (max3903) [inline account_payment_batch_process/models/account_payment.py]:
```suggestion
```

**Review by max3903 — CHANGES_REQUESTED** [review]

File: account_payment_batch_process/readme/CONTRIBUTORS.rst [hunk 4]
```diff
@@ -0,0 +1,4 @@
+* Balaji Kannan <bkannan@ursainfosystems.com>
+* Bhavesh Odedra <bodedra@ursainfosystems.com>
+* Sandeep Mangukiya <smangukiya@ursainfosystems.com>
+* Serpent Consulting Services Pvt. Ltd. <support@serpentcs.com>
```
Comment (max3903) [inline account_payment_batch_process/readme/CONTRIBUTORS.rst]:
```suggestion
* Murtuza Saleh <murtuza.saleh@serpentcs.com>
```

File: account_payment_batch_process/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2017 Ursa Information Systems <http://www.ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Batch Payments Processing",
+    "summary": "Process Payments in Batch",
+    "version": "11.0.1.0.1",
+    "license": "AGPL-3",
+    "author": "Ursa Information Systems, Odoo Community Association (OCA)",
```
Comment (max3903) [inline account_payment_batch_process/__manifest__.py]:
```suggestion
    "author": "Open Source Integrators, Odoo Community Association (OCA)",
```

File: account_payment_batch_process/__manifest__.py [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2017 Ursa Information Systems <http://www.ursainfosystems.com>
```
Comment (max3903) [inline account_payment_batch_process/__manifest__.py]:
```suggestion
# Copyright 2019 Open Source Integrators <https://www.opensourceintegrators.com>
```

File: account_payment_batch_process/models/__init__.py [hunk 7]
```diff
@@ -0,0 +1,4 @@
+# Copyright 2017 Ursa Information Systems <http://www.ursainfosystems.com>
```
Comment (max3903) [inline account_payment_batch_process/models/__init__.py]:
```suggestion
# Copyright 2019 Open Source Integrators <https://www.opensourceintegrators.com>
```

File: account_payment_batch_process/models/account_payment.py [hunk 8]
```diff
@@ -0,0 +1,109 @@
+# Copyright 2017 Ursa Information Systems <http://www.ursainfosystems.com>
```
Comment (max3903) [inline account_payment_batch_process/models/account_payment.py]:
```suggestion
# Copyright 2019 Open Source Integrators <https://www.opensourceintegrators.com>
```

File: account_payment_batch_process/wizard/invoice_batch_process.py [hunk 9]
```diff
@@ -0,0 +1,501 @@
+# Copyright 2017 Ursa Information Systems <http://www.ursainfosystems.com>
```
Comment (max3903) [inline account_payment_batch_process/wizard/invoice_batch_process.py]:
```suggestion
# Copyright 2019 Open Source Integrators <https://www.opensourceintegrators.com>
```

File: account_payment_batch_process/__manifest__.py (hunk 1 above)
Comment (murtuzasaleh) [reply account_payment_batch_process/__manifest__.py]: Why `12.0.1.0.0`? the code is migrated based on V11.0

**Review by max3903 — APPROVED** [review]

**Review by osi-scampbell — APPROVED** [review]

**Pull request merged**